### ISSUE TITLE
Does onnxruntime support bert with relative position embedding

### ISSUE THREAD

**Issue by trouble-maker007** [microsoft/onnxruntime#7713]
I have train a bert with relative position embedding, that improve the performance, I doubt that does onnxruntime support relative position embedding like [nezha](https://github.com/huawei-noah/Pretrained-Language-Model/tree/master/NEZHA-TensorFlow), [roformer](https://github.com/ZhuiyiTechnology/roformer)

**Comment by tianleiwu** [comment]
@trouble-maker007, are you able to export the model to onnx model? If so, let us know if onnxruntime cannot inference the model. 

I guess it only change the part of embedding layer, and the attention layers are not changed. So most our optimizations (like attention, layer normalization and GELU fusions) for BERT model can still be applied.

**Comment by RyanHuangNLP** [comment]
@tianleiwu thanks for response, maybe I will take a try to modify the transformer bert with relative position embedding to check whether able to export the model to onnx model
